newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -51,18 +60,28 @@ export default function SummaryPage() {
             </div>
           </div>
           <div className={styles['button-row']}>
+            <Button
+              icon
+              color="blue"
+              labelPosition="left"
+              disabled={mailSending || mailSent}
```

Comment:
What about adding `loading={mailSending}` to have the nice progress animation on the button while the request is running? Especially since sending emails can take a big depending on how many participants you have.